__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+beforeEach(() => {
+	return seed(testData);
+});
+
+afterAll(() => {
+	if (db.end) db.end;
```

Comment:
I think, this should be db.end() but im not 100% if you check the notes it should be in there :)